.appveyor/install.cmd

```diff
@@ -0,0 +1,63 @@
+@echo off
+setlocal enableextensions enabledelayedexpansion
+	cinst wget
+
+	if not exist "%PHP_BUILD_CACHE_BASE_DIR%" (
+		echo Creating %PHP_BUILD_CACHE_BASE_DIR%
+		mkdir "%PHP_BUILD_CACHE_BASE_DIR%"
+	)
+
+	if not exist "%PHP_BUILD_OBJ_DIR%" (
+		echo Creating %PHP_BUILD_OBJ_DIR%
+		mkdir "%PHP_BUILD_OBJ_DIR%"
+	)
+
+	if not exist "%PHP_BUILD_CACHE_SDK_DIR%" (
+		echo Cloning remote SDK repository
+		git clone --branch %SDK_BRANCH% %SDK_REMOTE% "%PHP_BUILD_CACHE_SDK_DIR%" 2>&1
```

Comment:
I'd suggest adding `-q --depth=1` to this `git clone`, though. I don't see any reason you need the full commit history, and `-q` will make the output a bit quieter. Since progress won't be reported to stderr, we can probably remove the redirection with `2>&1` as well.